tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_personalization_RTL.py

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This one fails on me as wel, using ar locale:
TEST-START | test_settings_personalization_RTL.py TestSettingsRTL.test_settings_app

Most recent errors/exceptions are:
08-03 13:49:38.780 I/Gecko   ( 1644): ###!!! [Parent][MessageChannel] Error: (msgtype=0x200081,name=PBrowser::Msg_Destroy) Channel error: cannot send/recv
08-03 13:49:39.690 E/Ringtones( 2312): [JavaScript Error: "TypeError: sendSyncMessage(...)[0] is undefined" {file: "chrome://marionette/content/listener.js" line: 434}]
08-03 13:49:41.330 W/Homescreen( 2429): Content JS WARN: L10nError: "collection-categoryId-undefined" not found in en-US in app://verticalhome.gaiamobile.org/index.html#1438602578918
08-03 13:49:41.340 W/Homescreen( 2429): Content JS WARN: L10nError: "collection-categoryId-undefined" not found in en-US in app://verticalhome.gaiamobile.org/index.html#1438602578918
08-03 13:49:41.340 W/Homescreen( 2429): Content JS WARN: L10nError: "collection-categoryId-undefined" not found in en-US in app://verticalhome.gaiamobile.org/index.html#1438602578918

The B2G process has restarted after crashing during  the tests so Marionette can't respond due to either a Gecko, Gaia or Marionette error. Above, the 5 most recent errors are listed. Check logcat for all errors if these errors are not the cause of the failure.
Failed to gather test failure debug.
Traceback (most recent call last):

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_client-0.16-py2.7.egg/marionette/runner/mixins/reporting.py", line 256, in gather_debug
    debug['source'] = self.marionette.page_source

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 1169, in page_source
    response = self._send_message('getPageSource', 'value')

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/decorators.py", line 36, in _
    return func(*args, **kwargs)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 715, in _send_message
    self._handle_error(response)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 751, in _handle_error
    raise errors.lookup(status)(message, stacktrace=stacktrace)

NoSuchWindowException: NoSuchWindowException: None


Failed to gather test failure debug.
Traceback (most recent call last):

  File "/Users/mwargers/B2G/gaia_clean/tests/python/gaia-ui-tests/gaiatest/runtests.py", line 70, in gather_debug
    marionette.switch_to_frame()

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 1283, in switch_to_frame
    return self._send_message("switchToFrame", "ok", **kwargs)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/decorators.py", line 36, in _
    return func(*args, **kwargs)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 715, in _send_message
    self._handle_error(response)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 751, in _handle_error
    raise errors.lookup(status)(message, stacktrace=stacktrace)

NoSuchWindowException: NoSuchWindowException: None


Failed to gather test failure debug.
Traceback (most recent call last):

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_client-0.16-py2.7.egg/marionette/runner/base.py", line 565, in gather_debug
    rv['source'] = marionette.page_source

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 1169, in page_source
    response = self._send_message('getPageSource', 'value')

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/decorators.py", line 36, in _
    return func(*args, **kwargs)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 715, in _send_message
    self._handle_error(response)

  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 751, in _handle_error
    raise errors.lookup(status)(message, stacktrace=stacktrace)

NoSuchWindowException: NoSuchWindowException: None


TEST-UNEXPECTED-ERROR | test_settings_personalization_RTL.py TestSettingsRTL.test_settings_app | IOError: Connection to Marionette server is lost. Check gecko.log (desktop firefox) or logcat (b2g) for errors.

Traceback (most recent call last):
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_client-0.16-py2.7.egg/marionette/marionette_test.py", line 296, in run
    testMethod()
  File "/Users/mwargers/B2G/gaia_clean/tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_personalization_RTL.py", line 48, in test_settings_app
    screen=manage_page.screen_element)
  File "/Users/mwargers/B2G/gaia_clean/tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py", line 255, in scroll
    smooth_scroll(marionette, screen, axis, vector, distance, increments)
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/gestures.py", line 43, in smooth_scroll
    action.perform()
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 452, in perform
    self.current_id = self.marionette._send_message('actionChain', 'value', chain=self.action_chain, nextId=self.current_id)
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/decorators.py", line 36, in _
    return func(*args, **kwargs)
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/marionette.py", line 691, in _send_message
    response = self.client.send(message)
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_transport-0.5-py2.7.egg/marionette_transport/transport.py", line 117, in send
    response = self.receive()
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_transport-0.5-py2.7.egg/marionette_transport/transport.py", line 73, in receive
    raise IOError(self.connection_lost_msg)
TEST-INFO took 497124ms

SUMMARY
-------
passed: 0
failed: 1
todo: 0

FAILED TESTS